diff --git a/dist/index.js b/dist/index.js
index 446463fd82c6228e98e9970ed9c3d499b22a9a24..8fa9a55a8f7992ad8b7a3a63bd8b20262af0c51e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -100,6 +100,11 @@ axios.interceptors.response.use(
     return Promise.reject(`Unexpected error calling BGG API: ${error.stack}`);
   }
 );
+axios.interceptors.request.use(function (config) {
+  const token = process.env.NEXT_PUBLIC_BGG_API_ACCESS_TOKEN;
+  config.headers.Authorization =  token ? `Bearer ${token}` : '';
+  return config;
+});
 
 // src/lib/helpers.ts
 var enforceArray = (value) => {
diff --git a/dist/index.mjs b/dist/index.mjs
index 04df80c36fe424d7a52d90126afc26d42d024c7c..78d8209d156ef201cf4dcf490ec5d88f70263cf8 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -67,6 +67,11 @@ axios.interceptors.response.use(
     return Promise.reject(`Unexpected error calling BGG API: ${error.stack}`);
   }
 );
+axios.interceptors.request.use(function (config) {
+  const token = process.env.NEXT_PUBLIC_BGG_API_ACCESS_TOKEN;
+  config.headers.Authorization =  token ? `Bearer ${token}` : '';
+  return config;
+});
 
 // src/lib/helpers.ts
 var enforceArray = (value) => {
